core/src/test/scala/com.mediative.amadou/sparkjob/TestEtl.scala

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
Copy of `test.TestEtl` to ensure compatibility with existing `SparkJob.run` definition